.flowconfig

```diff
@@ -16,7 +16,7 @@
 
 ; Ignore packages in node_modules which we (i.e. the jitsi-meet project) have
 ; seen to cause errors and we have chosen not to fix.
-.*/node_modules/babel-core/.*
```

Comment:
There was a problem running lint, we figure it out with @lyubomir and made that change :)